### DISCUSSION TITLE
Add "update" to older LXC

### DISCUSSION THREAD

**Discussion by Deekerman** [tteck/Proxmox#3143] [Ideas]
I currently have an older made SAB LXC from Tteck and i have to run the wget command to do any updates. It would be cool if you could retroactivly add the "update" command so i just have to type in update vs the whole wget. 

Thanks, 
D

**Comment by tteck** [comment]
Execute the following in the LXC console.
```
echo "bash -c \"\$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/sabnzbd.sh)\"" >/usr/bin/update
chmod +x /usr/bin/update
```

**Reply by Deekerman** [reply]
Thanks! that worked. So if i want to do that to any other script that is missing it, just swap out the wget part for the proper one?

**Reply by tteck** [reply]
yep, really just change the name `sabnzbd`
